modules/core/src/main/java/org/apache/ignite/client/ClientCompute.java

```diff
@@ -55,9 +55,23 @@
      * @return A Future representing pending completion of the task.
      * @throws ClientException If task failed.
      * @see ComputeTask for information about task execution.
+     * @deprecated Use {@link ClientCompute#executeAsync2(String, Object)}
```

Comment:
Maybe needs an explanation why is it deprecated.

Reply:
Fixed